code/game/objects/items/candle.dm

```diff
@@ -96,7 +96,7 @@
 	return 0
 
 /obj/item/candle/Crossed(var/obj/Proj)
-	if(isbeam(Proj))
+	if(isbeam(Proj) && level == LEVEL_ABOVE_FLOOR)
```

Comment:
instead of pasting this everywhere, it would be more OOP to add this to do this

```
/obj/item/Crossed(atom/movable/AM)
   if(level < LEVEL_ABOVE_FLOOR)
     return 1
   return 0
```
then for any of these items you can just add
```
if(..())
  return
```
at the top